packages/react-core/src/components/Text/examples/Text.md

```diff
@@ -9,201 +9,32 @@ propComponents: ['TextContent', 'Text', 'TextList', 'TextListItem']
 
 ### Headings
 
-```js
-import React from 'react';
-import {
-  TextContent,
-  Text,
-  TextVariants,
-  TextList,
-  TextListVariants,
-  TextListItem,
-  TextListItemVariants
-} from '@patternfly/react-core';
-
-<TextContent>
-  <Text component={TextVariants.h1}>Hello World</Text>
-  <Text component={TextVariants.h2}>Second level</Text>
-  <Text component={TextVariants.h3}>Third level</Text>
-  <Text component={TextVariants.h4}>Fourth level</Text>
-  <Text component={TextVariants.h5}>Fifth level</Text>
-  <Text component={TextVariants.h6}>Sixth level</Text>
-</TextContent>;
+```ts file="./TextHeadings.tsx"
```

Comment:
Since the examples are using `TextListVariants`, I would add it to the documentation so it shows up in the props table.  This can be done by adding it to the `propComponents` array on line 5 of this md file.

Reply:
`TextListVariants` does not render in the props table when added. This is likely because it is just an enum in `TextList`.

Reply:
@XiangyuShen you are correct!  We do not currently have a way to document enums.